db/migrate/20170222133503_change_back_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeBackColumnName < ActiveRecord::Migration[5.0]
```

Comment:
Since you haven't merged this into development you can just delete this migration and the one where you made the original change, there's no point in adding unnecessary migrations